### PR TITLE
Example for `apply_complex_function`

### PR DESCRIPTION
**Pull request by kolibril13**

### CODE REVIEW

**Comment by kolibril13** [comment]
> You should probably make this PR a draft for now :)

I think it is fixed now!

**Comment by AntonBallmaier** [comment]
Docs reference: https://manimce--1500.org.readthedocs.build/en/1500/reference/manim.mobject.mobject.Mobject.html#manim.mobject.mobject.Mobject.apply_complex_function

File: manim/mobject/mobject.py [hunk 1]
```diff
@@ -1146,6 +1146,32 @@ def apply_matrix(self, matrix, **kwargs):
         return self
 
     def apply_complex_function(self, function, **kwargs):
+        """Complex function that is applied to the xy-plane
+        With z= u+iv  u respresents the x value, v represents the y value.
+
+        Example
+        -------
+
+        ::manim ApplyFuncExample
```
Comment (AntonBallmaier) [inline manim/mobject/mobject.py:1155]:
```suggestion
        .. manim:: ApplyFuncExample
```

**Review by PhotonSpheres — APPROVED** [review]
LGTM

File: manim/mobject/mobject.py [hunk 2]
```diff
@@ -1245,6 +1245,32 @@ def apply_matrix(self, matrix, **kwargs):
         return self
 
     def apply_complex_function(self, function, **kwargs):
+        """Complex function that is applied to the xy-plane
```
Comment (Darylgolden) [inline manim/mobject/mobject.py:1248]:
```suggestion
        """Applies a complex function to a :class:`Mobject`.
```

File: manim/mobject/mobject.py [hunk 3]
```diff
@@ -1245,6 +1245,32 @@ def apply_matrix(self, matrix, **kwargs):
...
+        With z= u+iv  u respresents the x value, v represents the y value.
```
Comment (Darylgolden) [inline manim/mobject/mobject.py:1249]:
```suggestion
        The x and y coordinates correspond to the real and imaginary parts respectively.
```

**Review by Darylgolden — CHANGES_REQUESTED** [review]
Some phrasing changes I think would be good.

File: manim/mobject/mobject.py [hunk 4]
```diff
@@ -1245,6 +1245,32 @@ def apply_matrix(self, matrix, **kwargs):
...
+        .. manim:: ApplyFuncExample
+
+            class ApplyFuncExample(Scene):
+                def construct(self):
+                    circ = Circle().scale(1.5)
+                    circ_ref = circ.copy()
+                    circ.apply_complex_function(
+                        lambda x: np.exp(x*1j)
+                    )
+                    t=ValueTracker(0)
```
Comment (Darylgolden) [inline manim/mobject/mobject.py:1263]:
```suggestion
                    t = ValueTracker(0)
```

**Review by Darylgolden — CHANGES_REQUESTED** [review]

**Review by Darylgolden — APPROVED** [review]
LGTM

**Pull request merged**